ptvsd/wrapper.py

```diff
@@ -1726,6 +1746,14 @@ def start_server(port, addhandlers=True):
     server = _create_server(port)
     client, _ = server.accept()
     pydevd = _start(client, server)
+    def _wait_for_more():
+        while True:
+            client, _ = server.accept()
+            _add_socket_handler(client, pydevd, 'ptvsd.Server', killonclose=True, addhandlers=False)
+
+    connection_thread = threading.Thread(target=_wait_for_more, name='ptvsd_client_connection')
```

Comment:
New thread that'll ensure we support more than one client.
@int19h When speaking to Karthik the question about limiting multiple concurrent client connections came up.
Should we have a limit on the number of concurrent connections, to just one?

Reply:
Yes. We never supported anything more than that before, and just thinking of all the possibilities of things going wrong in that scenario makes my head hurt. :)